package.json

```diff
@@ -64,26 +64,27 @@
     "@babel/core": "^7.6.2",
     "@babel/runtime": "^7.6.2",
     "@react-native-community/bob": "^0.14.0",
-    "@react-native-community/cli": "^3.1.0",
-    "@react-native-community/cli-platform-android": "^3.1.0",
-    "@react-native-community/cli-platform-ios": "^3.1.0",
-    "@react-native-community/eslint-config": "^0.0.5",
+    "@react-native-community/cli": "^4.5.1",
+    "@react-native-community/cli-platform-android": "^4.5.1",
+    "@react-native-community/cli-platform-ios": "^4.5.0",
+    "@react-native-community/eslint-config": "^2.0.0",
     "babel-jest": "^26.5.2",
     "babel-plugin-module-resolver": "3.1.3",
     "detox": "16.7.2",
-    "eslint": "5.1.0",
-    "expo": "36.0.2",
-    "flow-bin": "0.105.2",
+    "eslint": "^6.0.0",
+    "expo": "^38.0.10",
```

Comment:
@Krizzu: Do you know how to test this? I couldn't make it work...

Reply:
We have `expo start:web` script to run web example - maybe try this?

Reply:
Finally got it running. Turns out I had a year old expo-cli installed 😛 

I hit other issues though that I can repro on `master`. When I run `start:web`, I get the following error message:

```
Module not found: Can't resolve '@react-native-async-storage/async-storage'
```

in both `async-storage/example/examples/GetSetClear.js` and `async-storage/example/examples/MergeItem.js`. When I tried changing the import to `../src/index`, I got a little further bit hit this instead:

```
Unhandled Rejection (TypeError): Cannot read property 'getItem' of undefined
GetSet.componentWillMount$
/~/example/examples/GetSetClear.js:29
  26 | };
  27 | 
  28 | async componentWillMount() {
> 29 |   const storedNumber = await AsyncStorage.getItem(STORAGE_KEY);
     |                                           ^
  30 |   if (storedNumber) {
  31 |     this.setState({
  32 |       storedNumber,
```

It seems like the web example got broken at some point. @EvanBacon: Is this error obvious to you?

Reply:
@Krizzu: If you don't mind, I think we should fix this separately. I've tested all the other platforms, and they're ready to go.

Reply:
Let's do it 🙏

Reply:
Filed #472